app/assets/javascripts/application.js

```diff
@@ -13,6 +13,7 @@
 //= require jquery
 //= require jquery_ujs
 //= require jquery.mobile.custom.min
+//= require jquery-ui.min
```

Comment:
We don't just copy dependencies into our code base instead we use gems. Please make use of https://github.com/joliss/jquery-ui-rails or